bcda/models/models.go

```diff
@@ -80,10 +81,22 @@ type ACOBeneficiary struct {
 	// Join model needed for additional fields later, e.g., AttributionDate
 }
 
+type CCLF9 struct {
```

Comment:
Do we need the xref so that we know if the number is an MBI or HICN?

Reply:
yessss